package test;

public class MyException extends Exception {

	public MyException() {
	System.err.println("Record already present Exception");
	}
	
}
